src/org/opensolaris/opengrok/web/api/v1/controller/RepositoriesController.java

```diff
@@ -38,7 +38,7 @@
 
     @GET
     @Path("/type")
-    @Produces(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.TEXT_PLAIN)
```

Comment:
I know that it's been like this before but please no for the future :D use json

Reply:
Well, I don't know what is exactly the rule. But if I'm returning just `String` which has no formatting then is it not better to return `TEXT/PLAIN`?

Reply:
Yes that is correct, but I mean in some future PR convert the API to json results.